smart_campus/app/models.py

```diff
@@ -227,9 +232,40 @@ def __str__(self):
             category=str(self.category)
         )
 
+    def get_primary_image(self):
+        """
+        Get the primary image
+        Return None when primary image didn't exist
+        """
+        primary_image = StationImage.objects.filter(
+            station=self,
+            is_primary=True
+        ).first()
+
+        if not primary_image:
+            return None
+        else:
+            return primary_image.image.url
```

Comment:
This code block is ok.
But I'll suggest the following way.

```python
if primary_image:
    return primary_image.image.url

return None
```

Even the last `return None` is not needed.